### PR TITLE
Devel

### PR DESCRIPTION
**Pull request by mridul2899**
I have added the code written till now.
It needs to be commented. It might also be needed to be formatted in a better manner.
It would be nice if this could be merged with the devel branch so that others can also work upon it if they want to.
Do suggest me the changes that you feel I must make before this code can get merged.

### CODE REVIEW

File: books/books.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import requests
+from bs4 import BeautifulSoup
+import os
+
+url = "http://gen.lib.rus.ec/search.php"
+search = (input("Enter the name of the book: "))
+r = requests.get(url, {'req':search})
+url = r.url + "&lg_topic=libgen&open=0&view=simple&res=100&phrase=1&column=def"
```
Comment (TheLethalCode) [inline books/books.py]: Instead of doing this like this, you could have given all of these in request parameters like the 'search'

**Comment by TheLethalCode** [comment]
Ok, looks good. But since this mainly command line oriented, it would be best if you partition the program into functions because it will be easier to integrate. And do not keep author name as mandatory. Ho and btw, always try using if __main__ == '__name__' . https://stackoverflow.com/questions/419163/what-does-if-name-main-do

**Comment by mridul2899** [comment]
Made the suggested changes. Ready to be merged.

**Comment by TheLethalCode** [comment]
LGTM :+1: The basic functionality of this has been taken care of. We will come back here, when we want to add some more features to it.

**Pull request merged**